src/interventions.c

```diff
@@ -534,12 +534,22 @@ void intervention_notify_contacts(
 			for( idx = 0; idx < n_contacts; idx++ )
 			{
 				contact = inter->individual;
-				if( contact->app_user )
-				{
-					if( inter->traceable == UNKNOWN )
-						inter->traceable = gsl_ran_bernoulli( rng, params->traceable_interaction_fraction );
-					if( inter->traceable )
+
+				// We first check if this interaction is manually traceable
+				if( inter->manual_traceable == UNKNOWN )
+					inter->manual_traceable = gsl_ran_bernoulli( rng, params->traceable_interaction_fraction );
```

Comment:
Shouldn't this be` manual_traceable_interaction_fraction`?

Also, another thing to consider is how best to model this in the context of manual contract tracing. If you see the same person on multiple days, do you have multiple chances to remember them? For the phone we decided yes as it we are modelling the fact someone was not carrying their phone. Also, is the value independent of the type of network? On the random network I suspect the value for manual contact tracing is very low, but on the occupational network it would be much higher.

Reply:
Completed fix - see discussion on latest pull request